cinema/tests/test_models.py

```diff
@@ -0,0 +1,410 @@
+import pytest
+
+from cinema.models import (
+    Country,
+    Genre,
+    ImdbRating,
+    MpaaRating,
+    Language,
+    Distributor,
+)
+
+country_names = ["USA", "UK", "France", "Italy", "Australia"]
+genre_names = ["Crime", "Drama", "Comedy", "Adventure", "Western"]
+imdb_ratings = [8.0, 8.5, 9.0]
+mpaa_ratings = ["PG-13", "R", "Not Rated"]
+mpaa_descriptions = [
+    "Children under 13", "Children under 17", "The upcoming movie"
+]
+language_names = ["English", "German", "Italian", "French", "Spanish"]
+distributor_names = [
+    "Columbia Pictures", "Paramount Pictures",
+    "Universal Pictures", "Warner Bros.", "Netflix"
+]
+
+
+@pytest.mark.django_db
+class TestCountryModel:
+    label_argvalues = [("name", "name"), ]
+    label_ids = [
+        f"'{arg[0]}' field: is label '{arg[1]}'?"
+        for arg in label_argvalues
+    ]
+    field_argvalues = [
+        ("name", "max_length", 64),
+        ("name", "unique", True),
+    ]
+    field_ids = [
+        f"'{arg[0]}' field: is {arg[1]} {arg[2]}?"
+        for arg in field_argvalues
+    ]
+
+    def test_record_is_instance_of_model(self, country):
+        assert isinstance(country, Country)
+
+    @pytest.mark.parametrize(
+        "field_name, expected_label",
+        label_argvalues,
+        ids=label_ids
+    )
+    def test_label_of_model_field(
+            self, country, field_name, expected_label
+    ):
+        actual_label = country._meta.get_field(field_name).verbose_name
+        assert actual_label == expected_label
+
+    @pytest.mark.parametrize(
+        "field_name, option_name, expected_option_value",
+        field_argvalues,
+        ids=field_ids
+    )
+    def test_options_of_model_fields(
+            self, country, field_name, option_name, expected_option_value
+    ):
+        if option_name == "max_length":
+            actual_option_value = country._meta.get_field(field_name).max_length
+        elif option_name == "unique":
+            actual_option_value = country._meta.get_field(field_name).unique
+        else:
+            raise AttributeError(
+                f"String '{option_name}' cannot be an attribute."
+            )
```

Comment:
Raise an exception in a test. It's nonsense. Tests do not raise exceptions at all. It's not business logic.